unix/main.c

```diff
@@ -416,7 +416,7 @@ int main(int argc, char **argv) {
             } else if (strcmp(argv[a], "-v") == 0) {
                 mp_verbose_flag++;
             } else if (strncmp(argv[a], "-O", 2) == 0) {
-                if (isdigit(argv[a][2])) {
+                if (isdigit((byte)argv[a][2])) {
```

Comment:
Use make V=1 or set BUILD_VERBOSE in your environment to increase build verbosity.
CC main.c
main.c: In function ‘main’:
main.c:419:17: error: array subscript has type ‘char’ [-Werror=char-subscripts]
                 if (isdigit(argv[a][2])) {
                 ^
cc1: all warnings being treated as errors
../py/mkrules.mk:47: recipe for target 'build/main.o' failed
make: *** [build/main.o] Error 1